src/learning-catalogue/model/factory/courseFactory.ts

```diff
@@ -26,12 +25,13 @@ export class CourseFactory {
 		course.id = data.id
 		course.description = data.description
 		course.learningOutcomes = data.learningOutcomes
+		course.preparation = data.preparation
 		course.shortDescription = data.shortDescription
 		course.title = data.title
 		course.modules = (data.modules || []).map(this._moduleFactory.create)
 		course.audiences = (data.audiences || []).map(this.audienceFactory.create)
+		course.status = ('status' in data) ? data.status : course.status
```

Comment:
I'm betting 10p this can be rewritten to 
```suggestion
		course.status = data.status || course.status
```

Reply:
I think one tests for the value of data.status and the other tests for the existence of a 'status' property in data.

Reply:
well if there's no `status` in `data` then `data.status` will be `undefined` and will evaluate to `false` with `||` and return `course.status` instead. I'll hold on to my 10p :-p It was good to learn about the `in` operator though.

Reply:
python wins 🐍